templates/single-tapestry.php

```diff
@@ -8,8 +8,39 @@
  *
  * @return object null
  */
+?>
+
+<?php
+$iframe_mode = array_key_exists('iframe', $_GET);
+$no_sidebar = array_key_exists('no-sidebar', $_GET);
+if ($iframe_mode):
+?>
+
+<!doctype html>
+<html lang="en-US">
+<head>
+    <meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link rel="profile" href="https://gmpg.org/xfn/11" />
+	<?php wp_head(); ?>
+    <style>
+        #wpadminbar {
+            display: none !important;
+        }
+        <?php if ($no_sidebar): ?>
+        .sidebar-container {
+            display: none !important;
+        }
+        <?php endif; ?>
+    </style>
+</head>
+<body <?php body_class(); ?>>
+<div id="page" class="site">
+<div id="content" class="site-content">
 
-get_header();
+<?php
```

Comment:
Rather than define the header content like this, it might be more eloquent to create an alternate header (maybe called iframe or frameless) and use the get_header function to call it. See the [Wordpress documenation](https://developer.wordpress.org/reference/functions/get_header/).

Reply:
The `get_header` function only looks at files in the **theme**, and after a bit of research, there is no way for it to use a custom header file in the **plugin**. Thus, I have changed to using the more low-level `load_template` which works similarly.